application/views/invoice_templates/pdf/InvoicePlane.php

```diff
@@ -0,0 +1,282 @@
+                <?php echo lang('subtotal'); ?>
+            </td>
+            <td class="text-right"><?php echo format_currency($invoice->invoice_item_subtotal); ?></td>
+        </tr>
+
+        <?php if ($invoice->invoice_item_tax_total > 0) { ?>
+            <tr>
+                <td <?php echo ($show_discounts ? 'colspan="5"' : 'colspan="4"'); ?> class="text-right">
+                    <?php echo lang('item_tax'); ?>
+                </td>
+                <td class="text-right">
+                    <?php echo format_currency($invoice->invoice_item_tax_total); ?>
+                </td>
+            </tr>
+        <?php } ?>
+
+        <?php foreach ($invoice_tax_rates as $invoice_tax_rate) : ?>
+            <tr>
+                <td <?php echo ($show_discounts ? 'colspan="5"' : 'colspan="4"'); ?> class="text-right">
+                    <?php echo $invoice_tax_rate->invoice_tax_rate_name . ' (' . $invoice_tax_rate->invoice_tax_rate_percent . '%)'; ?>
+                </td>
+                <td class="text-right">
+                    <?php echo format_currency($invoice_tax_rate->invoice_tax_rate_amount); ?>
+                </td>
+            </tr>
+        <?php endforeach ?>
+
+        <tr>
+            <td <?php echo ($show_discounts ? 'colspan="5"' : 'colspan="4"'); ?> class="text-right">
+                <b><?php echo lang('total'); ?></b>
+            </td>
+            <td class="text-right">
+                <b><?php echo format_currency($invoice->invoice_total); ?></b>
+            </td>
+        </tr>
+        <tr>
+            <td <?php echo ($show_discounts ? 'colspan="5"' : 'colspan="4"'); ?> class="text-right">
+                <?php echo lang('paid'); ?>
+            </td>
+            <td class="text-right">
+                <?php echo format_currency($invoice->invoice_paid); ?>
+            </td>
+        </tr>
+        <tr>
+            <td <?php echo ($show_discounts ? 'colspan="5"' : 'colspan="4"'); ?> class="text-right">
+                <b><?php echo lang('balance'); ?></b>
+            </td>
+            <td class="text-right">
+                <b><?php echo format_currency($invoice->invoice_balance); ?></b>
+            </td>
+        </tr>
+        </tbody>
+    </table>
+
+</main>
+
+<footer>
+    <?php if ($invoice->invoice_terms) : ?>
+        <div class="notes">
+            <b><?php echo lang('terms'); ?></b><br/>
+            <?php echo nl2br($invoice->invoice_terms); ?>
+        </div>
+    <?php endif; ?>
+</footer>
+
+<!-- IP-254 Receipts, start -->
+<?php
+foreach ($payments as $payment) {
+    if($payment->receipt_number) {
+        ?>
+        <header>
+            <h1 class="invoice-title"><?php echo lang('receipt') . ' ' . $payment->receipt_number; ?></h1>
+            <div id="company">
+                <div><b><?php echo $invoice->user_company; ?></b></div>
+                <?php if ($invoice->user_vat_id)
```

Comment:
Would be great to use the same format as line 19 for example. It's a template and most of users are not developers so it will be hard to understand the condition for them
```
if(condition) {
   echo "blabla";
}
```

Reply:
Needs label : 1.6 or 1.5.11 or something. Needs Milestone?